contracts/scripts/utils.js

```diff
@@ -31,110 +31,93 @@ async function main() {
   console.log('Token address:', token.address);
 }
 
-async function cancelTransaction(nonce) {
+// Helpers
+
+/**
+ * @param number nonce
+ * @param string gasPrice
+ * If a transaction is stuck, you can use this function to cancel that transaction as long as the gasPrice value is larger
+ */
+const cancelTransaction = async (nonce, gasPrice) => {
   const tx = await deployer.sendTransaction({
     to: config[NETWORK].CONTRACT_ADDRESS,
     value: 0,
     nonce: nonce,
-    gasPrice: ethers.utils.parseUnits('170', 'gwei'),
+    gasPrice: ethers.utils.parseUnits(gasPrice, 'gwei'),
     gasLimit: 100000,
   });
 
   console.log(tx);
-  // await tx.wait();
   console.log('done!');
-}
-
-async function createCohort(cohortId, limit, merkleRoot) {
-  const tx = await contract.createCohort(cohortId, limit, merkleRoot, {
-    gasPrice: ethers.utils.parseUnits('100', 'gwei'),
-  });
-  console.log(tx);
-  await tx.wait();
-}
+};
 
-async function fetchCohort(cohortId) {
+/**
+ * @param string cohortId
+ * Fetches the cohort on the contract and prints out the Limit number of the cohort
+ */
+const fetchCohort = async (cohortId) => {
   const tx = await contract.cohorts(cohortId);
   console.log(tx);
   console.log('LIMIT:', tx[0].toNumber());
-}
+};
 
-async function fetchUri(tokenId) {
+/**
+ * @param number tokenId
+ * Fetches the URI of the given tokenId
+ */
+const fetchUri = async (tokenId) => {
   const tx = await contract.tokenURI(tokenId);
   console.log(tx);
-}
-
-async function updateMerkleRoot(cohortId, merkleRoot) {
+};
+
+/**
+ * @param string cohortId
+ * @param string merkleRoot
+ * @param string gasPrice
+ * Updates the merkle root with new root value for a given cohort
+ */
+const updateMerkleRoot = async (cohortId, merkleRoot, gasPrice) => {
   const tx = await contract.setMerkleRoot(cohortId, merkleRoot, {
-    gasPrice: ethers.utils.parseUnits('50', 'gwei'),
+    gasPrice: ethers.utils.parseUnits(gasPrice, 'gwei'),
   });
   console.log(tx);
   await tx.wait();
-}
-
-async function createCohort(cohort_id, limit) {
+};
+
+/**
+ * @param string cohort_id
+ * @param number limit
+ * @param string gasPrice
+ * Creates a new cohort on the contract with an initial merkle root
+ */
+const createCohort = async (cohort_id, limit, gasPrice) => {
```

Comment:
consistency: camelCase used everywhere else, but not cohort_id

Reply:
nice!